src/test/obj_persist_count/out1.log.match

```diff
@@ -10,8 +10,8 @@ tx_alloc       4       1          1            0          1          0
 tx_alloc_next  4       1          1            0          1          0          1               0                 0               0                 3                      
 tx_free        1       1          1            0          0          0          0               0                 0               0                 1                      
 tx_free_next   1       1          1            0          0          0          0               0                 0               0                 1                      
-tx_add         534     14         8            0          4          1          5               3                 517             2                 15                     
-tx_add_next    4       3          1            0          1          0          1               1                 1               1                 4                      
+tx_add         4       3          1            0          1          1          1               0                 1               1                 2                      
+tx_add_next    4       3          1            0          1          1          1               0                 1               1                 2                      
```

Comment:
Out of curiosity: do you know where those 2 potential cache misses are located?

Reply:
I did investigate this, but concluded that we can't get rid of them.

I *think* it had to do with the data clobber method or some other transaction-scope state. The "tx_add_range" itself has no cache misses now.